scripts/index.js

```diff
@@ -1 +1,158 @@
-// add your solutions here
\ No newline at end of file
+// add your solutions here
+function positiveNumbers(ar) {
+    const newArray = []
+    ar.forEach(function(num){
+        if (num >= 0){
+            newArray.push(num);
+        }
+    })
+    return newArray;
+}
+console.log(positiveNumbers([1, 2, 4, -2, -1, 0, 5, -9, -6, -9.8, 8.7, 3]));
+
+//is positive with filter
+function positiveFilter(num){
+    if (num >= 0){
+        return num;
+    }
+}
+console.log("2nd variation of positive numbers using filter method.  this will not return zero")
+const numbers = [1, 2, 4, -2, -1, 0, 5, -9, -6, -9.8, 8.7, 3];
+console.log(numbers.filter(positiveFilter));
+
+function evenNumbers(ar) {
+    const newArray = []
+    ar.forEach(function(num){
+        if ((num % 2) === 0) {
+            newArray.push(num);
+        }
+    })
+    return newArray
+}
+console.log (evenNumbers([1, -4, 5, 2.2, -3, 0, 11, 10, 27, 3.3]));
+
+function squareTheNumbers(ar) {
+   return ar.map(function(num){
```

Comment:
Very nice way to avoid the extra variable and return the result of `.map` directly.